compiler/hash-tir/src/utils/params.rs

```diff
@@ -76,7 +76,7 @@ impl ParamError {
                         expected.len(),
                         pluralise!(expected.len())
                     ));
-                if let Some(location) = get_overall_location(*expected.value()) {
+                if let Some(location) = get_span(expected) {
```

Comment:
All `get_span` invocations can be replaced with a `.span()` + import of `HasNodeId`